crypto/Kconfig

```
@@ -23,6 +23,13 @@ if NRF_CC310_BL
 	default n # Only the no-interrupts version currently verified to work with Zephyr.
 endif
 
+menuconfig NRF_CC3XX_MBEDCRYPTO
+	bool "nrf_cc3xx_mbedcrypto SW crypto library for nRF devices with CryptoCell CC3xx."
+	depends on ENTROPY_CC3XX
+	help
+		It is not recommended to use the nrf_cc3xx_mbedcrypto without using
+		nrf_security as it is lacking some mbed TLS APIs
```

If this is promptless, then probably adjust the help text a bit to specify when / why you would enable this.

Something like:
```suggestion
	help
	  Use nrf_cc3xx_mbedcrypto library. This is automatically ensured when using nrf security
	  with the cc3xx backend.
	  Only select this setting without nrf security for special use cases, such as csrng support
	  without nrf security. 
```